mandatory/2-function-creation.js

```diff
@@ -15,15 +19,24 @@ Complete the function to check if the variable `num` satisfies the following req
 Tip: use logical operators
 */
 
-function validate(num) {}
+/*  function validate(num) {
+  if (typeof num === "number" && num % 2 === 0 && num <= 100){
+    return true; 
+  } else return false;
+}
+*/
+
+const validate = num => (typeof num === "number") && (num % 2 === 0) && (num <= 100) ? true : false;
```

Comment:
Here we don't need the `? true : false`.
Each boolean expression we have in the code will evaluate to either `true` or `false` and the combination of all these small checks is a boolean expression as well :)